src/transforms/remap.rs

```diff
@@ -69,6 +67,10 @@ impl TransformConfig for RemapConfig {
     fn transform_type(&self) -> &'static str {
         "remap"
     }
+
+    fn enable_concurrency(&self) -> bool {
+        true
+    }
```

Comment:
I think I'm failing to see why this is needed with the addition of `SynchronousTransform`. Are there synchronous transforms that could not be run concurrently?

Reply:
At least in theory, whether a transform is synchronous vs asynchronous has nothing to do with whether it's stateful vs stateless (or other ordering-related concerns). More practically, I wanted to limit the scope of this initial change and provide a simple on-ramp for the new functionality. 

On the other hand, you're right that this ought to apply safely to most (all?) current function-style transforms. I've gone through and added a bunch of common ones, which feels like a good place to start. It's my expectation that as we continue tweaking the transform design we'll have a good opportunity to go through each relevant transform, evaluate what's appropriate, and make sure the relevant features are enabled.

Reply:
👍 that's fine. I'm happy to see this evolve. Will `FunctionTransform` also go away once we transition all of the transforms over?

Reply:
I think it could potentially stick around as a convenience method for simple transforms or something similar, but I would like to separate thing like that from topology internals. Internally I'd like to collapse everything down to `SyncTransform`.